openstack/clustering/v1/policies/results.go

```diff
@@ -91,6 +98,17 @@ func (r *Policy) UnmarshalJSON(b []byte) error {
 		}
 	}
 
+	switch t := s.tmp.Spec.Version.(type) {
+	case float64:
+		if t == 1 {
+			r.Spec.Version = fmt.Sprintf("%.1f", t)
```

Comment:
I need this logic here to make sure that a version == 1 is returned as string "1.0"